packages/core/src/components/Review/ReviewLink.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class ReviewLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onClick) {
+      event.preventDefault();
+      this.props.onClick(this.props.href);
```

Comment:
To be consistent with other component's event handlers, I think we'd want to send the `event` as the first argument here. This would be consistent with the `Button` and `Tab` component (amongst others), and would allow a dev to do more advanced things if they wanted to.